README.md

```diff
@@ -1 +1,166 @@
+
+  ## 💻 Getting Started
+
+  To get a local copy up and running it in your machine, follow the next steps.
+
+  ### Prerequisites
+
+  In order to run this project you need:
+
+- Internet connection.
+- Basic knowledge of Git/Terminal.
+- Visual Studio Code Installed.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Install
+
+Install almost all dependencies.
+This one will install ESLint:
+
+> npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x @babel/plugin-syntax-jsx@7.x @babel/preset-react@7.x @babel/preset-react@7.x
+
+This one will install Stylelint:
+
+> npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project,clone the project from https://github.com/haftamudesta/Math-MagiciansTest
+
+> use the command git checkout component to go to the component dirctory
+> to create react use npm create vite@latest -- --template react
+> to install all raect dependencies use npm install after installing react
+> to run the code use npm run dev
+
+## 👥 Authors
+
+👤 **Haftamu Desta**
+
+- GitHub: [@haftamu](https://github.com/haftamudesta)
+- Twitter: [@DestaHaftamu](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
+- LinkedIn: [Haftamu Desta](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+
+👤 **Julio Quezada**
+
+- GitHub: [Alejandroq12](https://github.com/Alejandroq12)
+- Twitter: [@JulioAle54](https://twitter.com/JulioAle54)
+- LinkedIn: [Julio Quezada](https://www.linkedin.com/in/quezadajulio/)
+- 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features
+
+- [] **Make simple calculation**
+- [] **Read a random math-related quote**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support
+
+> Hi, I would appreciate if you give this project a star(⭐️) this will encourage me to improve it and build the social network of the future.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments
+
+> "Never stop learning, code never stop teaching"
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ
+
+- **Can I publish this project as mine?**
+
+  - No, but you can use it as an inspiration or for educational purposes.
+
+- **Can I modify it while practicing?**
+
+  - Yes, you can do it, but make sure that you know what are you doing, we never know when we can get stuck at some point just because we were playing with the code _-laughs-_
+
+   <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
This link is currently routing to a 404 page, please include the correct path to your license in order for it to behave correctly and to make the readme more effective.